SimTracker/SiStripDigitizer/plugins/PreMixingSiStripWorker.cc

```diff
@@ -339,6 +357,21 @@ void PreMixingSiStripWorker::put(edm::Event& e,
   edm::Service<edm::RandomNumberGenerator> rng;
   CLHEP::HepRandomEngine* engine = &rng->getEngine(e.streamID());
 
+  const bool simulateAPVInThisEvent = includeAPVSimulation_ && (CLHEP::RandFlat::shoot(engine) < fracOfEventsToSimAPV_);
+  float nTruePU = 0.;  // = ps.getTrueNumInteractions();
+  if (simulateAPVInThisEvent) {
+    iSetup.get<TrackerTopologyRcd>().get(tTopo);
+    iSetup.get<SiStripApvSimulationParametersRcd>().get(apvSimulationParametersHandle);
+    const auto it = std::find_if(
+        std::begin(ps), std::end(ps), [](const PileupSummaryInfo& bxps) { return bxps.getBunchCrossing() == 0; });
+    if (it != std::begin(ps)) {
+      nTruePU = it->getTrueNumInteractions();
```

Comment:
Since this is the main difference with the SiStripDigitizer implementation: is this the correct way to get the average number of interactions (Poisson parameter) for the in-time bunch-crossing? The original code is here: https://github.com/cms-sw/cmssw/blob/master/SimTracker/SiStripDigitizer/plugins/SiStripDigitizerAlgorithm.cc#L249-L281 but the same object is not available here (or I did not immediately find a way to get it)

Reply:
@pieterdavid In short: yes, it is.

(the `ps` is the `vector<PileupSummaryInfo>` from the premixed pileup event)

Reply:
@makortel thanks for confirming!